src/content/guidelines/accessibility/keyboard.md

```diff
@@ -1,28 +1,21 @@
+Keyboard accessibility is required and is helpful for users who rely on or prefer using a keyboard. Make sure to provide keyboard functionality to all interactive content. Common keyboard interactions include using the `Tab` key to select different interactive elements on a page and using the `Enter` key or the `Spacebar` to activate an in-focus element.
+
+[IBM Checkpoint 2.1.1 Keyboard](https://www.ibm.com/able/guidelines/ci162/keyboard.html)
 
 ### Focus indicators
-The `tab` key navigates through various interactive elements on a page. A default visual indicator is provided by the web browser in use. The display is an outlined border around the focused element. When an element is in focus, it can be further activated using the keyboard.
+The `tab` key navigates through all interactive elements on a page in the order they are written. A default visual indicator is provided by the web browser in use. The display is an outlined border around the focused element. When an element is in focus, it can be further activated using the keyboard.
 
-### Navigation order
-The order in which interactive elements receive focus is logical and predictable. Create the tab flow hierarchy by using the source code to arrange the keyboard navigation. Begin with the header, followed by the main navigation, then page navigation (from left to right, top to bottom), and end with the footer.
+[IBM Checkpoint 2.4.7 Focus Visible](https://www.ibm.com/able/guidelines/ci162/focus_visible.html)
 
-Use natively-accessible elements in navigation to activate links, buttons, and form controls with a keyboard. Reinforce semantic HTML to convey intent and meaning instead of simply defining the look and feel of an element. Enhance with ARIA (Accessible Rich Internet Application) labels when necessary. For additional customization, use `tabindex=“0”` to ensure a navigable element receives a focus indicator.
+### Navigation order
+The order in which interactive elements receive focus should be logical and predictable. Create the tab flow hierarchy by using the source code to arrange the keyboard navigation. A common flow might begin with the header, followed by the main navigation, then content navigation (from left to right, top to bottom), and end with the footer. Try to give all your users the same experience.
 
-Provide a “Skip to content” option when there is lengthy navigation so a user may access main content faster. This option is a link that is visually hidden by default and appears when it receives keyboard focus via the `tab` key.
+Use natively-accessible elements in navigation to activate links, buttons, and form controls with a keyboard. Reinforce semantic HTML to convey intent and meaning instead of simply defining the look and feel of an element. Enhance with ARIA (Accessible Rich Internet Application) labels when necessary. 
```

Comment:
`simply defining the look and feel of an element` could replace `simply` with solely, or something else to communicate how we shouldn't use HTML just for look and feel.